src/export_blob_metrics.py

```diff
@@ -35,16 +41,263 @@ def run_azcopy_ls(self) -> None:
                 n = 4 if k == "semifield-developed-images" else 2
                 outputtxt = Path(self.output_dir, k + ".txt")
                 # main azcopy list command
-                cmd = f'azcopy ls "{url + sas}" | cut -d/ -f 1-{n} | awk \'!a[$0]++\' > {outputtxt}'
+                cmd = f'./azcopy ls "{url + sas}" | cut -d/ -f 1-{n} | awk \'!a[$0]++\' > {outputtxt}'
```

Comment:
check for azcopy in the working directory or otherwise keep the azcopy without ./